Orange/widgets/data/owtable.py

```diff
@@ -884,6 +876,10 @@ def commit(self):
             # Selections of individual instances are not implemented
             # for SqlTables
             if isinstance(table, SqlTable):
+                summary = selected_data.approx_len() if selected_data else \
```

Comment:
`selected_data` is set to None at the beginning of the function and never set to anything else.